### ISSUE TITLE
[Question] Wine in Android

### ISSUE THREAD

**Issue by EliasTheGrandMasterOfMistakes** [AndreRH/hangover#148]
Wine can be builded for android but is very useless because of lack for emulate x86.

Hangover can be used in some way for build wine for android?

**Comment by AndreRH** [comment]
It can be used on android via termux. Checkout the readme. Consult the termux documentation on how to set up a graphical environment and install the hangover package

**Comment by EliasTheGrandMasterOfMistakes** [comment]
> It can be used on android via termux. Checkout the readme. Consult the termux documentation on how to set up a graphical environment and install the hangover package

oh thanks, have some plans for something like https://dl.winehq.org/wine-builds/android/? looks wine on android is abandonned
 https://archive.fosdem.org/2014/schedule/event/wine_android/

**Comment by AndreRH** [comment]
Hangover 0.4.0 release still has an APK for download. But it won't be revived.
See also https://github.com/AndreRH/hangover/issues/113

**Comment by EliasTheGrandMasterOfMistakes** [comment]
> Hangover 0.4.0 release still has an APK for download. But it won't be revived. See also #113

oh thanks, is sadly the android app in upstream wine is a legacy thing, i had hopes to someday all problems will be fixed in upstream and a future wine driver for android buffer display

**Comment by stefand** [comment]
The usefulness of Windows applications on Android has always been relatively limited due to the touchscreen vs mouse/keyboard interface. The main use for the Android version of Wine was on chromebooks, which luckily sometimes have x86 CPUs too. But afaiu nowadays chromebooks can run a proper Linux distro in chroot, mooting a lot of the Android specific code.

**Comment by EliasTheGrandMasterOfMistakes** [comment]
> The usefulness of Windows applications on Android has always been relatively limited due to the touchscreen vs mouse/keyboard interface. The main use for the Android version of Wine was on chromebooks, which luckily sometimes have x86 CPUs too. But afaiu nowadays chromebooks can run a proper Linux distro in chroot, mooting a lot of the Android specific code.

oh thanks

**Comment by Trass3r** [comment]
> The usefulness of Windows applications on Android has always been relatively limited due to the touchscreen vs mouse/keyboard interface.

That's true, but people go to great lengths. Even playing Age of Empires on a phone: https://www.reddit.com/r/EmulationOnAndroid/comments/1bxcigh/what_are_you_playing_on_winlator/

Newest development seems to be running things directly in termux to avoid the proot perf hit like in Winlator, see https://github.com/olegos2/mobox.

Hangover would further reduce the CPU load.
There's 1 guy who built it for termux but unfortunately not in any automated way:
https://github.com/alexvorxx/hangover-termux/releases

**Comment by AndreRH** [comment]
Check the packaging section of the readme. There's an official termux package for hangover

**Comment by Trass3r** [comment]
I see, thanks!
Unfortunately the documentation on that is rather incomplete. I had to activate the `tur-multilib` manually to be able to install hangover.
```sh
pkg install tur-repo
nano $PREFIX/etc/apt/sources.list.d/tur.list
pkg update
pkg install hangover-wine
```
